src/utils.rs

```diff
@@ -0,0 +1,14 @@
+#[inline(always)]
+pub(crate) fn get_home_dir() -> Option<String> {
+	#[allow(deprecated)]
```

Comment:
Every tool output is using different indentation sigh...
```suggestion
    #[allow(deprecated)]
```